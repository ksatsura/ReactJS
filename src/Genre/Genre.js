//imports
import React from 'react';
import '../style.css';

const Genre = (props) => {
    
    const genres = props.genres.join(' & ');

    return <p className='second-title'>{genres}</p>;
};

export default Genre;