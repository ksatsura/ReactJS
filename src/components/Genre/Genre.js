//imports
import * as React from 'react';
import '../../style.css';

// @flow
type Props = {
    genres: Array<string>
};

const Genre = (props: Props) => {
    
    const genres = props.genres.join(' & ');

    return <p className='second-title'>{genres}</p>;
};

export default Genre;