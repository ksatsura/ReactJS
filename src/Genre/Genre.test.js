import React from 'react';
import Genre from './Genre';
import {shallow} from 'enzyme';

describe('Genre.js', () => {

    const props = {
        genres: ['Comedy', 'Triller'],
    };

    describe('should render', () => {

        it('as expected', () => {
            const component = shallow(<Genre {...props} />);

            expect(component).toMatchSnapshot();
        });
    });
});