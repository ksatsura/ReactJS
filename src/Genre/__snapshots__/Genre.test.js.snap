// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genre.js should render as expected 1`] = `
<p
  className="second-title"
>
  Comedy & Triller
</p>
`;
